core/vm/opcodes.go

```diff
@@ -275,7 +276,7 @@ var opCodeToString = map[OpCode]string{
 	COINBASE:    "COINBASE",
 	TIMESTAMP:   "TIMESTAMP",
 	NUMBER:      "NUMBER",
-	DIFFICULTY:  "DIFFICULTY",
+	DIFFICULTY:  "DIFFICULTY", // TODO (MariusVanDerWijden) rename to RANDOM post merge
```

Comment:
Bleh, this is messy. So we put the random in the `Mixhash` header field, and alias the `RANDOM` over the `DIFFICULTY` opcode. Not much to do about it, I guess, just noting my displeasure :)

Reply:
Yep

Reply:
I'm unsure about a blind rename. IMHO we'll need to return one or the other depending on pre-post/merge.